packages/xrpl/src/index.ts

```diff
@@ -11,4 +11,6 @@ export * from './errors'
 
 export { default as Wallet } from './Wallet'
 
+export { keyToRFC1751Mnemonic, rfc1751MnemonicToKey } from './Wallet/rfc1751'
```

Comment:
Why is this exported?

Would it be better to export them as static methods on the wallet?

Reply:
I don't currently have any use for turning a secret into a mnemonic, but I figured if we're including the decode through `Wallet.fromMnemonic`, we should also offer the encode. (Especially since `rippled` has a slightly different implementation than RFC1751 elsewhere, so it'd require one line of custom code to use other solutions which is a pain)

Reply:
Wouldn't it be better to have a `.toMnemonic` method on the wallet then?

Reply:
In order to match our interface, `.toMnemonic` should probably allow them to generate bip39 mnemonics as well, which I don't want to implement currently.